etc/ReplayOfflineConfiguration.py

```diff
@@ -143,6 +143,8 @@
 # Setup repack and express mappings
 #
 repackVersionOverride = {
+    "CMSSW_10_1_10" : "CMSSW_11_1_0_pre8",
```

Comment:
Just for my curiosity: How come the first lines in these overrides are not overwritten to the `defaultCMSSWVersion`?

Reply:
I copied from #4581 (explanation should be in https://github.com/dmwm/T0/pull/4581#issuecomment-874581550)